packages/core/src/tree-view/tree-item.element.scss

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@import './../styles/tokens/generated/index';
+@import './../styles/mixins/mixins';
+
+:host {
+  --animation-duration: #{$cds-global-animation-duration-secondary};
+  --animation-easing: #{$cds-global-animation-easing-primary};
+  --background: #{$cds-alias-object-app-background};
+  --color: #{$cds-global-typography-color-400};
+  --focus-width: #{$cds-alias-object-border-width-300};
+  --font-size: #{$cds-global-typography-font-size-3};
+  --font-weight: #{$cds-global-typography-body-font-weight};
+  --icon-transform: rotate(90deg);
+  cursor: pointer;
+}
+
+.private-host {
+  cursor: inherit;
+
+  .item-children {
+    overflow: hidden;
+  }
+
+  .item-content {
+    background: var(--background);
+    border-left: var(--focus-width) solid transparent;
+    color: var(--color);
+    font-size: var(--font-size);
+    font-weight: var(--font-weight);
+  }
+
+  .item-content:hover {
+    --background: var(--cds-alias-object-interaction-background-hover);
+  }
+
+  .item-content:active {
+    --background: var(--cds-alias-object-interaction-background-active);
+  }
+
+  .lhs-container {
+    width: #{$cds-global-space-7};
+  }
+
+  .expand-collapse-icon {
+    --color: var(--cds-alias-object-interaction-color);
+    transform: var(--icon-transform);
+    transition: transform var(--animation-duration);
+  }
+}
```

Comment:
nit: Since the CSS is scoped could we remove this first group of nesting since it will generate excess CSS?